src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -75,4 +94,9 @@ public void periodic() {
         rpm.setDouble(this.shooterEncoder.getVelocity());
         rangeFilter.calculate(ballSensor.getRangeInches());
     }
+
+    public boolean checkTarget() {
```

Comment:
Because this returns a Boolean you should start it with `is` e.g `isAtTargetSpeed`. Helps a reader understand both what it is checking and also that it returns a Boolean where it's used in code